drivers/flash/Kconfig.cadence_qspi_nor

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2022 Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+DT_COMPAT_CAD_QSPI_NOR := cdns,qspi-nor
```

Comment:
Update to new convention.  See either drivers/flash/Kconfig.* for examples:

```
default y
depends on DT_HAS_CDNS_QSPI_NOR_ENABLED
```

Reply:
Resolved, Thanks.